files/postfix/userdb-aliases.cf

```diff
@@ -0,0 +1,6 @@
+domain = hashbang.sh
+hosts = postgresql://userdb-attempt-too-do-user-989073-0.db.ondigitalocean.com:25060/userdb?sslmode=require
```

Comment:
this needs to include `user:password@`, as discussed on IRC hosts with a URI does not work with the `user` and `password` settings.

Reply:
which also means that `user` and `password` below this needs to be removed, to reflect live changes that do work